src/github.com/getlantern/flashlight/analytics/analytics.go

```diff
@@ -25,7 +27,8 @@ const (
 )
 
 var (
-	log = golog.LoggerFor("flashlight.analytics")
+	log  = golog.LoggerFor("flashlight.analytics")
+	elog = errlog.ErrorLoggerFor("flashlight.analytics")
```

Comment:
With the new suggestion by @oxtoacart we can define `Error` in `errors` package, and extend `golog` to simply recognize it.